auth-web/src/store/activityLog.ts

```diff
@@ -0,0 +1,33 @@
+import { ActivityLog, ActivityLogFilterParams } from '@/models/activityLog'
```

Comment:
Travis, in the official Pinia documentation, they advised to changed the folder's name to "stores". It might be worthwhile doing that. I did that for Filings UI.

Reply:
I think it's `store` now, I'll change to `stores`